lms/djangoapps/learner_home/test_serializers.py

```diff
@@ -753,22 +785,28 @@ def test_happy_path(self):
             "courseRun",
             "gradeData",
             "certificate",
-            "enrollment"
+            "enrollment",
         ]
 
         assert output_data.keys() == set(expected_keys)
-        assert output_data['courseRun'] is None
-        assert output_data['gradeData'] is None
-        assert output_data['certificate'] is None
-        assert output_data['enrollment'] == UnfulfilledEntitlementSerializer.STATIC_ENTITLEMENT_ENROLLMENT_DATA
+        assert output_data["courseProvider"] is not None
+        assert output_data["courseRun"] is None
+        assert output_data["gradeData"] is None
+        assert output_data["certificate"] is None
+        assert (
+            output_data["enrollment"]
+            == UnfulfilledEntitlementSerializer.STATIC_ENTITLEMENT_ENROLLMENT_DATA
+        )
 
     def test_static_enrollment_data(self):
         """
         For an unfulfilled entitlement's "enrollment" data, we're returning a static dict.
         This test is to ensure that that dict has the same keys as returned by the LearnerEnrollmentSerializer
         """
         output_data = TestEnrollmentSerializer().serialize_test_enrollment()
-        expected_keys = UnfulfilledEntitlementSerializer.STATIC_ENTITLEMENT_ENROLLMENT_DATA.keys()
+        expected_keys = (
+            UnfulfilledEntitlementSerializer.STATIC_ENTITLEMENT_ENROLLMENT_DATA.keys()
```

Comment:
also unnecessary

Reply:
Not fighting on this. I `black` everything.